core/src/test/scala/unit/kafka/integration/PartitionLoggingTest.scala

```diff
@@ -70,4 +110,20 @@ class PartitionLoggingTest extends IntegrationTestHarness{
       }
     }, "the partition's UnderMinISR state should be " + shouldBeUnderMinISR)
   }
+
+  private def checkPartitionOfflineStatus(offlinePartitionExist: Boolean, tp: TopicPartition): Unit = {
```

Comment:
nit: would "expectOfflinePartitionExists" be a more accurate variable name? (otherwise it sounds like you're declaring that one is definitely offline, which is what the method itself tests, right?)

Reply:
checkPartitionOfflineStatus is "check partition offline status". It means checking the status. I could change to "checkOfflinePartitionExists" if this sounds more accurate to you.

Reply:
The method name is fine; it says exactly what it does. But the variable sounds like "an offline partition exists," which is not what it means: the other argument is the one and only partition being checked, and the boolean is about whether that one partition is expected to be online or offline during this check. Something like "shouldBeOffline" or "expectOfflineStatus" is more accurate and less confusing.

That said, it's a minor naming issue in a test's helper method, so up to you.